apps/loan/loan-products/src/app/shared/services/loan-products.service.ts

```diff
@@ -37,7 +47,33 @@ export class LoanProductsService {
     creditors
   );
 
-  constructor() {}
+  public pingStore = this.graphSvc.createEntityStore<any>({
+    autoLoad: false,
+    getQuery: PingQueryDocument,
+  });
+
+  public plaidStore = this.graphSvc.createEntityStore<LinkQuery>({
+    autoLoad: false,
+    getQuery: LinkDocument,
+  });
+
+  constructor(
+    private graphSvc: GraphQLStoreCreatorService,
+    public dialogService: DialogService,
+    public storage: AppStorageService
+  ) {
+    this.pingStore.getData().subscribe();
```

Comment:
Without having looked to hard, does this need to unsubscribe somewhere?

Reply:
It's a cold observable like an http call.